django-stubs/utils/decorators.pyi

```diff
@@ -16,10 +13,6 @@ def method_decorator(decorator: Callable | Iterable[Callable], name: str = ...)
 def decorator_from_middleware_with_args(middleware_class: type) -> Callable: ...
 def decorator_from_middleware(middleware_class: type) -> Callable: ...
 def make_middleware_decorator(middleware_class: type[MiddlewareMixin]) -> Callable: ...
-
-class AsyncGetResponseCallable(Protocol):
-    def __call__(self, __request: HttpRequest) -> Awaitable[HttpResponseBase]: ...
```

Comment:
This was just added in #1425, I don't think we should be removing it?

Reply:
Renamed it to `_AsyncGetResponseCallable` with regard to @sobolevn recommendation, and moved it in `deprecation.pyi`, where I added a usage.

I must say I don't have the full context for #1425 and #1269, so I'd be happy to rollback the changes and leave it for later.

Reply:
This is my mistake: adding public protocols is not a good idea: because they do not exist t runtime. We must not promote importing them.

As far as I understand, this protocol is only used in our `django.utils` package.
So, it should be protected.

Reply:
Thanks for cleaning this up, I had already forgotten.